src/index.html

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE html>
+Hello, world!
```

Comment:
You forget to add base HTML markup to your code. 
Type `!` in the empty file and press the `tab` button and see what will happen)

Also, you need to add an empty line at the end of your code